modules/cluster-logging-collector-log-forwarding-supported-plugins.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-log-forwarding-supported-plugins_{context}"]
+
+= Supported log data output types
+
+[role="_abstract"]
+Red Hat OpenShift Logging version 5.0 provides the following output types and protocols for sending log data to target log collectors.
+
+Red Hat tests each of the combinations shown in the following table. However, you should be able to send log data to a wider range target log collectors that ingest these protocols.
+
+[options="header"]
+|====
+| Output types   | Protocols          | Target log collector
```

Comment:
"Tested with" rather than "Target" - we don't want to imply these are the only versions we can forward to, just that they're the ones we've used in testing.

Reply:
Thanks, @alanconway. I've updated the heading. Please mark this PR as "/lgtm"

Reply:
/lgtm